examples/server/server.py

```diff
@@ -170,6 +170,21 @@ async def on_ended():
     )
 
 
+async def add_candidate(request):
+    params = await request.json()
+
+    pc_id = request.match_info["pc_id"]
+    pc = pcs[pc_id]
```

Comment:
Shouldn't there be some code to generate a 404 error if the ID is unknown?

Reply:
Ultimately this sample is not secure anyway, so I thought getting rid of cares for abnormal cases might be helpful as a whole making it easier for the readers to understand the entire code flow?

Reply:
But yea it depends on the policy of the samples in this repo.
Will add the handler.